client/src/router/AppRouter.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { createBrowserRouter, createRoutesFromElements, Route } from 'react-router-dom';
+import { ROUTES, RouteType } from './AppRoutes';
+import { Router } from '@remix-run/router';
+
+const router: Router = createBrowserRouter(
```

Comment:
This builds the router object based on the routes specified in `AppRoutes.tsx`. 

The benefit of this implementation is that as we add new routes this code remains unchanged.